rojak-pantau/rojak_pantau/spiders/metrotvnews.py

```diff
@@ -62,38 +82,60 @@ def parse_news(self, response):
         self.logger.info('parse_news: {}'.format(response))
         is_video = response.css('ul.breadcrumb > li > a::text').extract()[0] == 'VIDEO'
 
-        # Skip if video page, since no author here
-        if is_video:
-            return
-
         # Init item loader
         # extract news title, published_at, author, content, url
+        # Required: title, raw_content, published_at
         loader = ItemLoader(item=News(), response=response)
         loader.add_value('url', response.url)
 
-        title = response.css('div.part.lead.pr > h1::text').extract()[0]
-        info = response.css('div.part.lead.pr > span::text').extract()[0]
-        author_name = info.split('-')[0].strip()
+        # Will be dropped if video page
+        if is_video:
+            return loader.load_item()
+
+        title_selectors = response.css('div.part.lead.pr > h1::text')
+        if not title_selectors:
+            # Will be dropped on the item pipeline
+            return loader.load_item()
+        title = title_selectors.extract()[0]
+        loader.add_value('title', title)
+
+        raw_content_selectors = response.css('div.part.article')
```

Comment:
`$x('//div[@class="part article"]/node()
  [not(
    descendant-or-self::comment()|
    descendant-or-self::script|
    descendant-or-self::div|
    descendant-or-self::iframe)]')`

```
[u'\r\n        ',
 u'<p><strong>Metrotvnews.com, Jakarta</strong>: Komisi Pemilihan Umum (KPU) DKI Jakarta membatasi penggunaan dana kampanye pada Pilgub DKI 2017. Sumber dana pasangan calon hanya boleh datang dari pasangan calon, partai, dan pihak lain yang sah menurut hukum.<br><br>\r\nKomisioner KPU DKI Jakarta Bidang Pencalonan dan Kampanye Dahlia Umar menjelaskan, pembatasan dana kampanye terdiri dari pembatasan pemasukan dan pembatasan pengeluaran. Sumber dana bisa berbentuk uang, barang, dan jasa.\r\n                </p>',
 u'\r\nSumber dana kampanye yang berasal dari perseorangan tidak boleh melebihi Rp75 juta. Kelompok dan atau badan hukum swasta paling banyak Rp750 juta. Sumbangan dari parpol maksimal Rp750 juta. Hanya sumber dana dari paslon yang tidak dibatasi jumlahnya.',
 u'<br>',
 u'<br>',
 u'\r\n"Di undang-undang, yang dibatasi bukan maksimal penerimaan sumbangan, tapi maksimal yang disumbangkan oleh penyumbang," jelas Dahlia dalam acara sosialisasi kampanye dan dana kampanye Pilgub DKI 2017 di Hotel Mercure, Jalan Cikini Raya, Jakarta Pusat, Selasa (18/10/2016).',
 u'<br>',
 u'<br>',
 u'\r\nPembatasan pengeluaran dana kampanye adalah akumulasi dari empat kegiatan. Pertama, pengeluaran rapat umum, yakni jumlah peserta dikali frekuensi kegiatan dikali standar biaya daerah.',
 u'<br>',
 u'<br>',
 u'\r\nKedua, pengeluaran pertemuan terbatas, yakni jumlah peserta dikali frekuensi kegiatan dikali standar biaya daerah.',
 u'<br>',
 u'<br>',
 u'\r\nKetiga, pertemuan tatap muka, yakni jumlah peserta dikali frekuensi dikali standar biaya daerah. Pembuatan bahan kampanye dengan perhitungan jumlah kegiatan dikali (30% dikali jumlah pemilih) dikali Rp25.000.',
 u'<br>',
 u'<br>',
 u'\r\nAlat peraga kampanye yang dibiayai paslon jumlahnya berpedoman pada Keputusan KPU Provinsi/KIP Aceh atau KPU/KIP Kabupaten. "Jadi, secara otomatis sumbangan berapa pun yang diterima calon pada akhirnya akan dibatasi pengeluarannya," imbuh dia.',
 u'<br>',
 u'<br>',
 u'\r\nKPU DKI tidak mengatur mekanisme jika terjadi kelebihan dana kampanye. Komisioner KPU DKI Bidang Sosialisasi Betty Idroos mengatakan, pihaknya akan berkoordinasi terkait kelebihan dana kampanye dengan KPU RI.',
 u'<br>',
 u'<br>',
 u'\r\n"Itu ada kekosongan (hukum), tidak diatur, dan sepanjang pengetahuan saya, sudah berkoordinasi dengan KPU RI sisa itu belum tahu akan dikemanain, apakah dikembakikan kepada kas negara atau dapat dikelola oleh tim setelah terpilih," kata Betty.',
 u'<br>',
 u'<br>',
 u'\r\n',
 u' \xa0',
 u'<br>',
 u'<br>',
 u'\r\n\r\n        ',
 u'<br>',
 u'(MBM)\r\n\t\t        \r\n    ']
```

itu paragraf pertama emang di tag `p` yang terpisah gitu, coba cek lagi. ini hasil di atas diambil dari: http://m.metrotvnews.com/pilkada/news-pilkada/1bVYDO2N-kpu-dki-batasi-dana-kampanye

gimana?